sections/combat.tex

```diff
@@ -241,6 +247,17 @@ \subsection*{\pagetarget{Walls}{Defending a Town With\\a Citadel\raggedright}}\i
 \vspace{1em}
 \note{9}{In a siege against AI, the Arrow Tower follows the priority-rules like a Silver \svg{silver} Unit. But if there are many valid targets, it cannot attack the nearest one, so it will attack the one that is closest to perishing (the smallest difference between \svg{health_points-yellow} and current \svg{damage-yellow}).}
 
+\subsection*{Alternative Combat Boards}
+\begin{expansion}{navalbattles}
+	As an \textbf{optional rule} you can use the Naval Battle Board for combats at sea locations. At this combat board the two middle spaces are removed, so that Ground \svg{unit_ground} units must navigate around these spaces and Flying \svg{unit_flying} Units can move over them, but still cannot end their movement there. 
```

Comment:
```suggestion
	As an \textbf{optional rule} you can use the Naval Battle Board for combats at sea locations.
	At this combat board the two middle spaces are removed, so that Ground \svg{unit_ground} units must navigate around these spaces and Flying \svg{unit_flying} Units can move over them, but still cannot end their movement there. 
```